sabnzbd/nzbstuff.py

```diff
@@ -1511,6 +1522,14 @@ def check_first_article_availability(self):
                     return False
         return True
 
+    def check_missing_threshold_mbytes(self):
+        """ Return false if more than 50% is missing after downloading missing_threshold_mbytes MB """
+        if self.bytes_missing > self.bytes_downloaded:
+            missing_threshold = cfg.missing_threshold_mbytes() * MEBI
+            if missing_threshold and self.bytes_tried > missing_threshold:
+                return False
```

Comment:
Maybe add `INFO` logging here to notify why the job was aborted.

Reply:
The other tests will only do a debug log. How about either a debug or info log for each test, like this?
`logging.debug('Abort job "%s", due to impossibility to complete it (test: check_first_article_availability)', self.final_name)`